index.html

```diff
@@ -169,7 +194,7 @@ <h2 id="resources"><a href="#resources">Research, visualizations, talks...</a></
             <li>
             <a href="http://datahackermd.com/2013/language-use-on-github/">Language Use on GitHub</a> visualization of the rank correlation between languages.</li>
             <li><a href="http://www.fastcolabs.com/3015178/the-top-10-hottest-github-projects-right-now?partner=rss">The Top 11 Hottest GitHub Projects Right Now</a></li>
-            <li>"New repos &amp; most watched" <a href="https://github.com/hnq90/GitHubArchive">email reports</a> (<a href="http://us6.campaign-archive1.com/?u=ecbe15df98da933fc8209cf1b&id=31fc9cb07c">example</a>). Signup: <a href="http://eepurl.com/bb4EFL">daily</a>, <a href="http://eepurl.com/bc5JV9">every 3 days</a>, <a href="http://eepurl.com/bc5Yn9">weekly</a>.</li>
+            <li><a href="http://thechangelog.com/nightly/">Subscribe to Changelog Nightly</a> (the new and improved GitHub Archive daily email reports). It ships every night at 10pm CT &mdash; and unearths the hottest new repos on GitHub before they blow up. It's nerd to the core and in your inbox each night.</li>
```

Comment:
Let's keep this one as is, albeit we should probably add a note indicating that its a set of alternative reports.. Want to give credit where its due :)